Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -4,6 +4,19 @@ import MapboxDirections
 
 let NavigationEventTypeRouteRetrieval = "mobile.performance_trace"
 
+/**
+ The `EventsManagerFreeDriveDataSource` protocol declares values required for recording free drive events.
+ */
+public protocol EventsManagerFreeDriveDataSource: AnyObject {
```

Comment:
Consider a more generic name for this protocol, like “RawLocatable” or “RawLocationDataSource”.

Reply:
This protocol contains values that we want to report in passive navigation mode, it can be updated in the future to contain other fields as well. However, I renamed the `FreeDrive` part to the `PassiveNavigation`.